### PR TITLE
FIX Inserting a 'Download a file' without selecting an image

### PR DESCRIPTION
**Pull request by mikenz**
In the html editor, when inserting a 'Download a file' link without selecting a file to link to, no link should be inserted. Currently is inserting a link to an undefined file id: [file_link,id=undefined]

### CODE REVIEW

**Comment by dhensby** [comment]
great catch - is this only happening in master or is it a problem in 3?

**Comment by mikenz** [comment]
I don't run 3 so don't know, sorry.

**Comment by dhensby** [comment]
Wow, nice. Thanks for running v4, it's great to see it getting an outing in the real world.

**Comment by tractorcow** [comment]
Thanks, but since this is in a src file, can you please run the node compiler over this to build the dist file?

`npm install && npm run build` should do it. ;)

**Comment by mikenz** [comment]
Updated dist files, of course this will cause merge conflicts with some of my other pull requests for whoever is merging.

**Comment by dhensby** [comment]
merge on green ticks

**Comment by dhensby** [comment]
closed with 37f11f3

**Comment by dhensby** [comment]
thanks!

File: javascript/dist/i18n.js [hunk 1]
```diff
@@ -132,7 +132,7 @@
 						return match;
 					}
 
-					return subMatch1 + params[i += 1];
+					return subMatch1 + params[i++];
```
Comment (tractorcow) [inline javascript/dist/i18n.js]: I think I forgot to ask someone to build dist files in a prior PR... oh well this fixes it!

**Pull request closed without merging**